x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/AutoConfigureNode.java

```diff
@@ -659,16 +660,30 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
                         || localFinalEnv.settings().hasValue(NetworkService.GLOBAL_NETWORK_BIND_HOST_SETTING.getKey())
                         || localFinalEnv.settings().hasValue(NetworkService.GLOBAL_NETWORK_PUBLISH_HOST_SETTING.getKey()))) {
                         bw.newLine();
-                        bw.write(
-                            "# With security now configured, which includes user authentication over HTTPs, "
-                                + "it's reasonable to serve requests on the local network too"
-                        );
+                        bw.write("# Allow HTTP API connections from localhost and local networks");
+                        bw.newLine();
+                        bw.write("# Connections are encrypted and require user authentication");
                         bw.newLine();
                         bw.write(
-                            HttpTransportSettings.SETTING_HTTP_HOST.getKey() + ": " + httpHostSettingValue(NetworkUtils.getAllAddresses())
+                            HttpTransportSettings.SETTING_HTTP_HOST.getKey() + ": " + hostSettingValue(NetworkUtils.getAllAddresses())
                         );
                         bw.newLine();
                     }
+                    if (false == (localFinalEnv.settings().hasValue(TransportSettings.HOST.getKey())
+                        || localFinalEnv.settings().hasValue(TransportSettings.BIND_HOST.getKey())
+                        || localFinalEnv.settings().hasValue(TransportSettings.PUBLISH_HOST.getKey())
+                        || localFinalEnv.settings().hasValue(NetworkService.GLOBAL_NETWORK_HOST_SETTING.getKey())
+                        || localFinalEnv.settings().hasValue(NetworkService.GLOBAL_NETWORK_BIND_HOST_SETTING.getKey())
+                        || localFinalEnv.settings().hasValue(NetworkService.GLOBAL_NETWORK_PUBLISH_HOST_SETTING.getKey()))) {
+                        bw.newLine();
+                        bw.write("# Allow other nodes to join the cluster from localhost and local networks");
+                        bw.newLine();
+                        bw.write("# Connections are encrypted and mutually authenticated");
+                        bw.newLine();
+                        bw.write("#" + TransportSettings.HOST.getKey() + ": " + hostSettingValue(NetworkUtils.getAllAddresses()));
```

Comment:
Does this trigger the bootstrap checks, which by default fail on linux?

Reply:
It;s commented out ;)